Backend/lib/yudhishthira/src-read-only/Lib/Yudhishthira/Storage/Queries/AppDynamicLogic.hs

```diff
@@ -0,0 +1,68 @@
+{-# OPTIONS_GHC -Wno-dodgy-exports #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-# OPTIONS_GHC -Wno-unused-imports #-}
+
+module Lib.Yudhishthira.Storage.Queries.AppDynamicLogic (module Lib.Yudhishthira.Storage.Queries.AppDynamicLogic, module ReExport) where
```

Comment:
hm did not found, where these queries used?

Reply:
these are autogenerated 🙃